Nuget/Ploeh.AutoFixture.nuspec

```diff
@@ -6,6 +6,9 @@
         <title>AutoFixture</title>
         <authors>Mark Seemann</authors>
         <owners>Mark Seemann</owners>
+        <dependencies>
+            <dependency id="Albedo" version="[1,2)" />

```

Comment:
I choose Albedo 1.0.1 simply because it was already referenced in [unit tests](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixtureUnitTest/packages.config#L3). I overlooked that the dependency is added to the publicly visible AutoFixture package which should reference the _lowest possible version_ (1.0.0).

I'll get back on this!